packages/p2p/src/components/orders/order-table/order-table-row.jsx

```diff
@@ -25,7 +25,7 @@ const Title = ({ send_amount, currency, order_purchase_datetime, order_type }) =
     );
 };
 
-const OrderRow = ({ style, row: order }) => {
```

Comment:
Just for my understanding, we don't need this property, am I right?

Reply:
yes, it seems we don’t use it inside table.row